thinkhazard/static/less/report.less

```diff
@@ -229,3 +229,25 @@ body {
     margin-bottom: 4px;
   }
 }
+
+.style-pdf-overview-btn {
+  padding: 7px 10px;
+  margin-top: 15px;
+  border: 1px solid #d9d9d9;
+  font-size: 14px;
+  .border-top-radius(5px);
+  .border-right-radius(5px);
+  .border-left-radius(5px);
+  .border-bottom-radius(5px);
+  &:hover {
+    .box-shadow(5px 5px 5px rgba(0, 0, 0, 0.15));
+  }
+}
+
+.btn-xs {
+  .style-pdf-overview-btn;
+}
+
+.btn-default {
+  .style-pdf-overview-btn;
+}
```

Comment:
Are you sure you want to change .btn-default as well?
.btn-default is not meant to be used for sizes, only for colors